charts/geoweb-taf-backend/values.yaml

```diff
@@ -73,6 +130,28 @@ taf:
       limits:
         memory: "100Mi"
         cpu: "1"
+#    livenessProbe:
+#      httpGet:
+#        path: /healthcheck
+#        port: 8085
+#      initialDelaySeconds: 5
+#      periodSeconds: 5
+#      failureThreshold: 5
+#    readinessProbe:
+#      httpGet:
+#        path: /health_check?readiness=true
+#        port: 8085
+#      initialDelaySeconds: 5
+#      periodSeconds: 5
+#      failureThreshold: 1
```

Comment:
why is this commented out?

Reply:
Same situation for tafplaceholder as above publishers